src/makeRobotAccountant.js

```diff
@@ -6,7 +6,20 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+  const firstThreeCalls = 3;
```

Comment:
You can remove this variable and just add 3 as compare property into ternary operator